config/class-site-details-index.php

```diff
@@ -103,6 +111,38 @@ public function get_plugin_info() {
 		return $plugin_info;
 	}
 
+	/**
+	 * Gather basic information about VIP Search for a site
+	 */
+	public function get_search_info() {
+		$search_info = array();
+
+		$search_info['enabled'] = ( defined( 'USE_VIP_ELASTICSEARCH' ) && USE_VIP_ELASTICSEARCH ) ||
+			( defined( 'VIP_ENABLE_VIP_SEARCH' ) && true === VIP_ENABLE_VIP_SEARCH );
+
+		$search_info['query_integration'] = ( defined( 'VIP_ENABLE_ELASTICSEARCH_QUERY_INTEGRATION' ) && true === VIP_ENABLE_ELASTICSEARCH_QUERY_INTEGRATION ) ||
```

Comment:
This logic is encapsulated in `Search::is_query_integration_enabled()` so we can re-use that.

Reply:
For maximum flexibility and understanding, what if we named this `query_integration_enabled`? Or made it an array - `'query_integration' => array( 'enabled' => $enabled )`.

Reply:
Using `Search::is_query_integration_enabled()` looks much better! Taking that into account, I was wondering if it would make sense to detect whether VIP is enabled by checking if the `Search` class exists, as it's loaded from `000-vip-init.php` precisely on that condition.

I like the `query_integration_enabled` option, I'll change it.

Reply:
I'm finally checking for the `Search` class. I've also added a comment in the `mu-plugins` loader so in change the loading logic changes, whoever is in change remembers to update the Site Details generation.

Reply:
And I've also changed the Jetpack details generation. Now there's an 'available' field that identifies if the Jetpack code is loaded at all.